app/views/devise/registrations/show.html.erb

```diff
@@ -1,35 +1,43 @@
 <%- model_class = User -%>
-<div class="page-header">
-  <h1><%=@user.name %></h1>
-</div>
+<%= stylesheet_link_tag 'users' %>
 
-<br>
-<div class="text-center">
-<%= image_tag @user.avatar_url || image_path('missing_avatar.png'), id: 'preview', style: 'max-width:500px; max-height:500px'%>
+<div class="container text-center">
+  <div class="page-header">
+    <h1><%= @user.name %></h1>
+  </div>
+  <div class="row">
+    <%= image_tag @user.avatar_url || image_path("missing_avatar.png"), id: 'preview', class: 'profile-image'%>
+  </div>
+  <div class="row">
+    <% if @user.birthday.present? %>
+        <p class="description-tag"><%= model_class.human_attribute_name(:birthday) %></p>
+        <p class="description-text"><%= @user.birthday.strftime("%d.%m.%Y") %></p>
+    <% end %>
+  </div>
+  <div class="row">
+    <% if @user.telephone_number.present? %>
+        <p class="description-tag"><%= model_class.human_attribute_name(:telephone_number) %></p>
+        <p class="description-text"><%= @user.telephone_number %></p>
+    <% end %>
+  </div>
+  <div class="row">
+    <% if @user.telegram_username.present? %>
+        <p class="description-tag"><%= model_class.human_attribute_name(:telegram_username) %></p>
+        <p class="description-text"><%= @user.telegram_username %></p>
+    <% end %>
+  </div>
+  <div class="row">
+    <% if @user.favourite_sports.present? %>
+        <p class="description-tag"><%= model_class.human_attribute_name(:favourite_sports) %></p>
+        <p class="description-text"><%= @user.favourite_sports %></p>
+    <% end %>
+  </div>
+  <div class="row">
+    <% if can? :edit, @user %>
+        <%= link_to t('.edit', :default => t('navbar.drop-down.edit_profile')),
+                    edit_user_path(@user), :class => 'btn btn-default btn-save-edit-user btn-centered' %>
```

Comment:
It seems all properties of `btn-default` are overwritten so it's safe to remove.